content/password-cracking-and-how-to-protect-against-them/index.md

```diff
@@ -0,0 +1,187 @@
+
+You can strengthen your passwords using these steps: 
+1. **Make them long**
+* At least 16 characters, but longer passwords are more secure.
+2. **Make them random**
+* Use a random string of lowercase and uppercase letters, numbers, and symbols.
+* You can create a memorable  phrase of 4-7 unrelated words called “passphrase”. Passphrases are often used in physical cryptowallets (example: **Caterpillar Orange Socks Running Ocean Beard Coffeemug**). You can use spaces to strengthen the passphrase, as shown in the example.  
+3. **Make them unique**
+* Use a different password for every account. 
+* Don't reuse your password, please. 🙏
+
+> You can test the strength of your password using tools like [**PasswordMonster**](https://www.passwordmonster.com/), to see how long a password may take to crack. As a precaution, don’t actually put your real password in. Your password is just for you, and you should never blindly trust websites. 
+
+### Multi-Factor Authentication (MFA) 💳
+**Multi-Factor Authentication (MFA)** is an authentication method that requires more than two forms of identity verification before allowing a user access to an account or network.
+
+Popular additional verifications are: 
+* One-time passcodes (OTPs)
+* Time-based one-time passcodes (TOTP)
+* Biometric verification
+* Push notifications
+
+> MFA can often be confused with 2FA (Two-Factor Authentication), because they are similar. 2FA can be considered a type of MFA, requiring just two verification methods instead of one. If you want to read more about the differences between them we have an [article](https://supertokens.com/blog/mfa-best-practices) covering that. 
+
+### Password Managers 🗝️
+
+Remembering strong passwords is not possible. Especially since you need a different one for every account you have. 🤯  
+
+That’s why you can use a **password manager**, like 1Password or LastPass, to store them. A password manager is an app on your phone or computer that stores your passwords for you in an encrypted vault.
+
+Another benefit that password managers provide is that they can generate random passwords for you so you won’t even need to come up with them yourself. 👏
+
+> **Bonus:** They will even let you know if you have reused a password or if your password is weak. How nice!
+
+### Rate-Limiting and Lockout Mechanisms 🛑
+**Rate-limiting** is a technique that controls the number of requests that can be made to a network. It’s used to protect systems against malicious attacks like Denial-of-Service, and from overloading a system.
+
+**Lockout mechanism**  is a security feature that locks a user’s account after a certain number of incorrect password attempts. Accounts are usually locked after 3 to 5 unsuccessful attempts to login. It’s used to prevent brute force and automated attacks.
+
+### Monitoring Accounts and Alerting Users 🚨
```

Comment:
From around 2020 onwards, there have been several session hijacking attacks on YouTube, where attackers would hijack Youtubers accounts by stealing their accounts and through privilege escalation lock the owners out of them and run bitcoin scam adverts. Suspicious activity alerting would prevent attacks such as this. Also SuperTokens detects session hijacking which we can also mention

Reply:
Will look into this and add it, while I wait for the cover :)